### PR TITLE
Fix port validation check

### PR DESCRIPTION
**Pull request by christopherarter**
Change validation check logic to prevent a port number of `0` being evaluated as false.

### CODE REVIEW

**Comment by freekmurze** [comment]
Thanks! Could you also add a test that makes sure that this works?

**Comment by christopherarter** [comment]
@freekmurze  test added!

File: src/Ssh.php [hunk 1]
```diff
@@ -111,7 +111,7 @@ protected function getExtraOptions(): string
             $extraOptions[] = "-i {$this->pathToPrivateKey}";
         }
 
-        if ($this->port) {
+        if ($this->port > -1) {
```
Comment (freekmurze) [inline src/Ssh.php]: I'm thinking that this code now allows the invalid port `-1` to be used.

Comment (deleugpn) [reply src/Ssh.php]: This works fine (https://3v4l.org/ufUIW) but I would agree that `>= 0` reads better than `-1` in this context

Comment (freekmurze) [reply src/Ssh.php]: > This works fine

Ah, right I was mistaken. Yeah, best change it to `>=` for better readability. Thanks!

Comment (christopherarter) [reply src/Ssh.php]: comparison changed to `>= 0`

**Comment by freekmurze** [comment]
Tests are failing for this one, so I'm going to close this in favour of another PR.

Thanks for your work on this!

**Pull request closed without merging**